components/data-list/css/_mixin.scss

```diff
@@ -19,18 +19,17 @@
   color: var(--utrecht-data-list-item-key-color);
   font-size: var(--utrecht-data-list-item-key-font-size);
   font-weight: var(--utrecht-data-list-item-key-font-weight);
-  grid-area: label;
   line-height: var(--utrecht-data-list-item-key-line-height);
 }
 
 @mixin utrecht-data-list-item-value {
   /* configure `min-block-size` in case the value is empty */
-  color: var(--utrecht-data-list-label-color);
+  color: var(--utrecht-data-list-item-value-color, var(--utrecht-data-list-label-color));
```

Comment:
We can simply remove the `--utrecht-data-list-label-color`, the `label` was a bug.